Resources/Public/JavaScript/kitodopublication.js

```diff
@@ -1945,13 +1966,14 @@ var listProjectHtml = function (id, all = '') {
     return projectHtml;
 }
 
-var listHtml = function (name, id, all = '', optionalText = '', color = '') {
+var listHtml = function (name, id, all = '', optionalText = '', status = '') {
     JSON.stringify(all).replace(/"/g, '');
 
-    if (color) {
-        colorHtml = '('+color+')';
+    var statusHtml = '';
+    if (status) {
+        statusHtml = '('+status+')';
     } else {
-        colorHtml = '';
+        statusHtml = '';
```

Comment:
The else-branch can be deleted since it's redundant to the initialization of the `statusHTML`-variable.